lib/src/tag/tagger.dart

```diff
@@ -184,8 +186,8 @@ class Tagger {
       if (_isBinaryOnly) {
         tags.add(Sdk.dart.tag);
         explanations.add(Explanation('Binary only',
-            'Cannot assign flutter SDK tag because it is binary only',
-            tag: null));
+            'Cannot assign Flutter SDK tag because it is binary only.',
```

Comment:
We should probably also reconsider this...

A binary-only package can usually be used as a tool with the flutter sdk...

Reply:
I'm all for this! It seems to be better to be optimistic with the scope / capabilities of such packages, and if we get reports that something should be otherwise, we should add checks for conflicts later on...